src/structs/2captcha.ts

```diff
@@ -1,18 +1,4 @@
-import fetch from "../utils/fetch"
-import { APIError } from "./2captchaError"
-import * as utils from "../utils/generic"
-import  getProviderData  from "./providers/providers"
-import { softId } from "./constants/constants"
-import checkCaptchaParams from "../utils/checkCaptchaParams"
-
-
-const provider = getProviderData ()
-
-interface BaseSolve {
```

Comment:
This has no effect whatsoever at runtime as it's empty and it's generally unused, which is why I removed it.

Reply:
Hi. Yes indeed the BaseSolve interface does not make sense. We will remove it in the next release.